packages/tools/src/utilities/viewport/scrollToSlice.ts

```diff
@@ -0,0 +1,60 @@
+import { Types, getEnabledElement, StackViewport } from '@cornerstonejs/core';
+
+type Metadata = {
```

Comment:
The type needs to be exposed.

Reply:
Can we only have the actually used properties on here for now? We can add more when they are actually used.